composer.json

```diff
@@ -35,12 +35,12 @@
     },
     "require-dev": {
         "mongodb/builder": "^0.2",
-        "laravel/scout": "^11",
+        "laravel/scout": "^10.3",
```

Comment:
Version 10.3 required for `whereNotIn` feature that we implement. https://github.com/laravel/scout/commit/9b45a429a54d9ad1c2117e4fb4fcc75a2a0cacf9